gapic-generator/templates/default/service/rest/service_stub/grpc_transcoding_method/_def.erb

```diff
@@ -10,17 +10,19 @@
 #   Uri, Body, Query string parameters
 def <%= method.rest.transcoding_helper_name %> request_pb
   transcoder = Gapic::Rest::GrpcTranscoder.new
+  <%- method.http_bindings.each do |binding| -%>
```

Comment:
I suggest calling this variable `http_binding` to avoid confusion (and possible subtle bugs) since `binding` is a method call on Kernel that is often used in templating systems.

Reply:
:+1: